### DISCUSSION TITLE
node-js-learning-through-making-build-a-cpu-sensor/

### DISCUSSION THREAD

**Discussion by giscus** [thisdavej/comments-website#2] [Announcements]
# node-js-learning-through-making-build-a-cpu-sensor/

We are back with our LTM (Learning through Making) tutorials and ready to hit the ground running and write some real Node.js code!  In this series, we will learn about Node.js in the context of creating IoT (Internet of Things) projects.  We will build a “CPU Sensor” in this first project since CPU loading/utilization is a “sensor” we can measure, record, and ultimately stream to other locations.  In future tutorials, we will harness the power of Node.js to interact with physical sensors that live outside of our computing environment.

https://thisdavej.com/node-js-learning-through-making-build-a-cpu-sensor/

**Comment by thisdavej** [comment]
test
